### PR TITLE
docs: update content-security-policy

### PR DESCRIPTION
**Pull request by lindenquan**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos
- [ ] Infra

# Description

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have ran `pnpm change` and documented my changes
- [x] I have made corresponding changes to the Qwik docs
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/(qwikcity)/advanced/content-security-policy/index.mdx [hunk 1]
```diff
@@ -17,7 +17,7 @@ created_at: '2023-06-15T11:45:23Z'
 
 [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) is an added layer of security that helps to detect and mitigate certain types of attacks, including Cross-Site Scripting (XSS) and data injection attacks. These attacks are used for everything from data theft, to site defacement, to malware distribution.
 
-## Steps to integrate
+## Steps to integrate (Doesn't work for SSG)
```
[the remark this answers is not in the document]
Comment (gioboa) [reply packages/docs/src/routes/docs/(qwikcity)/advanced/content-security-policy/index.mdx:20]: Yep, in fact, onRequest runs at build time for SSG.
Can we add a clearer explanation?
something like this:
> If you are using the Server Side Generation approach, you don't have an up and running server, so this code will not run during the page request.

**Review by gioboa — CHANGES_REQUESTED** [review]
Thanks for your help @lindenquan 
I added my feedback.

**Comment by lindenquan** [comment]
> Thanks for your help @lindenquan I added my feedback.

Thanks @gioboa  for quick response. I have update the documents. Please let me know if there are anything else to update.

**Review by gioboa — APPROVED** [review]
Thanks @lindenquan

**Pull request merged**